pennylane/ops/qubit/other.py

```diff
@@ -0,0 +1,187 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module submodule contains the discrete-variable quantum operations that
+do not fit nicely in any of the other categories.
+"""
+import functools
+
+# pylint:disable=abstract-method,arguments-differ,protected-access
+import numpy as np
+
+import pennylane as qml
+from pennylane.operation import AnyWires, Observable, Operation
+from pennylane.wires import Wires
+
+
+class Projector(Observable):
```

Comment:
Wondering: would it make sense to create a separate file just for dedicated observable subclasses? `Hermitian` and `SparseHamiltonian` could go there too.

It should be okay to have operators that subclass both from `Operation` and `Observable` in one place (e.g., as non parametric ops).

Reply:
Well... I did have an "observable" file, but then I got rid of it due to @thisac now outdated comment. Maybe I should recreate it?

Reply:
Oops, I'm sorry about that. I'm happy either way, although I was slightly leaning towards not having an observable file without e.g. the Paulis in it (so that's where my suggestion earlier came from). I think both solutions work though.

Reply:
How about leaving a comment about the Paulis & Hadamard being in the other file? That way we could have a result in the file, should anyone search there.